entries/u/umbrella.com.json

```
@@ -0,0 +1,17 @@
+{
+  "Umbrella": {
+    "domain": "umbrella.com",
+    "url": "https://umbrella.cisco.com",
+    "tfa": [
+      "sms",
+      "custom-software"
+    ],
+    "custom-software": [
+      "Duo"
```

```suggestion
      "sms"
```
Whilst the Duo app is mentioned in the documentation, it doesn't look like any Duo-specific authentication methods are supported. Therefore, the custom software attribute isn't needed.

---

Thanks for your correction! ❤️